config/config.go

```diff
@@ -50,6 +50,8 @@ var (
 		`"`, `\"`,
 		`\`, `\\`,
 	)
+
+	updatedConfigPath = "./updated_config.conf"
```

Comment:
Is this just for testing?

Reply:
we want to save the config to another path so this should be here, we can change if we want though?